home/templates/home/question_detail.html

```diff
@@ -0,0 +1,242 @@
+{% extends "home/base.html" %}
+{% load static %}
+{% block content %}
+
+
+<style>
```

Comment:
I'd recommend having a shared, static CSS file - its better for browser-side caching.

Reply:
We planned to move all styles to shard CSS file after everyone will merge their templates, so won't be conflicts in the shared file

Reply:
You'll get code duplication that way... If some parts of your UI have separate, distinct styles, you can put them in separate CSS files.

Reply:
I moved all the CSS to different file